composer.json

```diff
@@ -13,7 +13,8 @@
     ],
     "require": {
         "php": ">=5.6",
-        "wp-cli/wp-cli": "^2"
+        "wp-cli/wp-cli": "^2",
+        "inmarelibero/gitignore-checker": "dev-master"
```

Comment:
I think we should lock to a specific version before we merge this: https://github.com/inmarelibero/gitignore-checker/pull/11#issuecomment-1717635762

Reply:
Done.